dbt/adapters/firebolt/connections.py

```diff
@@ -104,10 +104,9 @@ def get_response(cls, cursor: SQLConnectionManager) -> AdapterResponse:
         fields. code field is missing for Firebolt adapter, as it's not returned
         from the SDK/API.
         """
-        success = 'False'
+        success = 'SUCCESS'
```

Comment:
Won't this return `SUCCESS` even if the query fails?

Reply:
Yes.

@stepansergeevitch , correct me if I'm wrong, but I don't think the SDK returns a message on failure. Instead it just errors out.

Reply:
Then you can just 
```
        return AdapterResponse(
            _message='SUCCESS',
            rows_affected=cursor.rowcount,
            code=None,
        )
```

Reply:
Good point. That extra variable is a now-unnecessary historic relic.